src/Sarif.Sdk.sln

```diff
@@ -81,7 +81,9 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Test.UnitTests.Sarif.Multit
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Sarif.WorkItemFiling", "Sarif.WorkItemFiling\Sarif.WorkItemFiling.csproj", "{5D8A3D57-3593-40E2-A7DF-0B4E182543B6}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Test.UnitTests.Sarif.WorkItemFiling", "Test.UnitTests.Sarif.WorkItemFiling\Test.UnitTests.Sarif.WorkItemFiling.csproj", "{4162EF3A-B3B0-45AD-99DC-C833C73494C9}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Test.UnitTests.Sarif.WorkItemFiling", "Test.UnitTests.Sarif.WorkItemFiling\Test.UnitTests.Sarif.WorkItemFiling.csproj", "{4162EF3A-B3B0-45AD-99DC-C833C73494C9}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Test.EndToEnd.Baselining", "Test.EndToEnd.Baselining\Test.EndToEnd.Baselining.csproj", "{8D674F7B-F6D8-4508-A940-AA7D03D5DA36}"
```

Comment:
>EndToEnd [](start = 58, length = 8)

Note to self: Do we have a convention for distinguishing "functional" tests (_e.g._, `Test.FunctionalTests.Sarif`) from "end to end" tests? It might become clear as I read on.

Reply:
Oh, I see, it's a whole separate executable (`Program.cs`). Now I'm curious how and when we run such tests.

---
In reply to: [366488718](https://github.com/microsoft/sarif-sdk/pull/1734#discussion_r366488718) [](ancestors = 366488718)

Reply:
... and now the comment in `Program.cs` answers my question.

---
In reply to: [366489311](https://github.com/microsoft/sarif-sdk/pull/1734#discussion_r366489311) [](ancestors = 366489311,366488718)